### DISCUSSION TITLE
When is the next version due?

### DISCUSSION THREAD

**Discussion by denivic** [mindee/doctr#1451] [General]
When will we get 0.8?

**Comment by felixdittrich92** [comment]
Hi @denivic :wave:,

We actually wanted to release it at the beginning of January, but we had some difficulties retraining the DB models. As soon as issue #1364  is resolved we will release 0.8.0.

**Reply by denivic** [reply]
Sounds good! Thanks for the update 🔥

**Comment by felixdittrich92** [comment]
CC @odulcy-mindee (I got lots of this questions already :sweat_smile:)

**Comment by odulcy-mindee** [comment]
Hello @denivic ! :wave:
As @felixdittrich92 said, as soon as db models are done, we'll do a new release of docTR! At the time being, you need to install `main` branch to use latest features and checkpoints from docTR.

Thank you for the quick update @felixdittrich92
